services/uploads/src/auditUploads.ts

```diff
@@ -0,0 +1,114 @@
+import { Context, S3Event } from 'aws-lambda';
+import { rm, stat, readdir } from 'fs/promises'
+import path from 'path'
+
+import { NewS3UploadsClient, S3UploadsClient } from './s3'
+
+import { NewClamAV, ClamAV } from './clamAV'
+
+export async function auditUploadsLambda(event: S3Event, _context: Context) {
```

Comment:
Thanks for including this! It was the first thing I thought of when I read your pings this morning, and you already did it ❤️ .

Do we want to set this on a cron schedule for the lambda? There's an example of that serverless config in the `cleanup` lambda in `app-api`. Couldn't hurt to have a semi-regular full bucket av scan happening.

Reply:
that’s a good idea. Need to figure out how for it to report errors to us then